editor.planx.uk/src/pages/PlatformAdminPanel/utils.ts

```diff
@@ -1,15 +1,13 @@
+import { capitalize } from "lodash";
 import { AdminPanelData } from "types";
 
-export const internalTeamNames = [
-  "WikiHouse",
-  "PlanX",
-  "Open Systems Lab",
-  "Testing",
-  "Open Digital Planning",
-  "Environment Agency",
-  "Templates",
-];
+export const getFlowNamesForFilter = (data: AdminPanelData[]) => {
```

Comment:
This helper function was previously in a `.tsx` file (but it's not a component), I've simply moved it to `utils.ts` instead here